src/app/components/hris/employees/employee-profile/accordions/accordion-documents/accordion-additional-documents/accordion-additional-documents.component.ts

```diff
@@ -30,6 +30,7 @@ export class AccordionDocumentsCustomDocumentsComponent {
   }
 
   fileCategories = [];
+  unarchivedCustomDocuments: any = [];
```

Comment:
Refactor: I know this is all over the code, however, we should consider distinguishing between global/member vars and local. Would help with readability.

member: m_unarchivedCustomeDocumets
local: unarchivedCustomeDocumets

Reply:
I will have to discuss with the team about naming conventions for global and local variables. Since this is something we not currently doing.